ebpf/cmd/playground/main.go

```diff
@@ -3,75 +3,327 @@
 package main
 
 import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
 	"fmt"
 	"os"
-	"strings"
+	"strconv"
 	"time"
 
+	"github.com/bufbuild/connect-go"
 	"github.com/go-kit/log"
+	ebpfmetrics "github.com/grafana/pyroscope/ebpf/metrics"
+	"github.com/pkg/errors"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/model/relabel"
+
+	"github.com/go-kit/log/level"
+	pushv1 "github.com/grafana/pyroscope/api/gen/proto/go/push/v1"
+	"github.com/grafana/pyroscope/api/gen/proto/go/push/v1/pushv1connect"
+	typesv1 "github.com/grafana/pyroscope/api/gen/proto/go/types/v1"
 	ebpfspy "github.com/grafana/pyroscope/ebpf"
+	"github.com/grafana/pyroscope/ebpf/pprof"
 	"github.com/grafana/pyroscope/ebpf/sd"
 	"github.com/grafana/pyroscope/ebpf/symtab"
 	"github.com/grafana/pyroscope/ebpf/symtab/elf"
 	"github.com/prometheus/client_golang/prometheus"
+	commonconfig "github.com/prometheus/common/config"
+)
+
+var configFile = flag.String("config", "", "config file path")
```

Comment:
super cool do you have an config file example ?

Reply:
Nevermind I see the default !

Reply:
I've used locally like this. This is only for "testing/prototyping" locally.  
```
{
  "RelabelConfig": [
    {
      "SourceLabels": [
        "__meta_process_exe"
      ],
      "Regex": ".*python.*",
      "Action": "keep"
    },
    {
      "SourceLabels": [
        "__meta_process_cwd"
      ],
      "TargetLabel": "service_name",
      "Regex": ".*pythonProjectAsync.*",
      "Replacement": "python/ProjectAsync",
      "Action": "replace"
    },
    {
      "SourceLabels": [
        "__meta_process_cwd"
      ],
      "TargetLabel": "service_name",
      "Regex": ".*examples/python/rideshare/flask.*",
      "Replacement": "python/flask/rideshare",
      "Action": "replace"
    },
    {
      "SourceLabels": [
        "service_name"
      ],
      "Regex": "python/.*",
      "Action": "keep"
    }
  ]
}
```

We could create something better if we ever decide to put it into profilecli